scripts/insert_into_appointments.sql

```diff
@@ -1 +1,7 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date, time, doctor_id, patient_id)
+  VALUES ('2016-08-12','10:00 am', 4, 8),
+         ('2016-08-17','1:00 pm', 6, 5),
+         ('2016-08-17','10:00 am', 6, 8),
+         ('2016-08-17','1:00 pm', 6, 4)
```

Comment:
Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables.